.github/workflows/build.yml

```diff
@@ -1,6 +1,7 @@
 name: build
 
 on:
+  workflow_dispatch:
```

Comment:
Can you remind me what this does?

Reply:
Oh, I was trying to test an addition to this pull request. It was on a pr of this... pull request. I added this while trying to do test the addition.
This allows for manual testing - but only if the default branch (```master```) has that too.